pkgs/development/libraries/AntTweakBar/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, unzip, xlibs, mesa }:
+
+stdenv.mkDerivation rec {
+  name = "AntTweakBar-1.16";
+
+  buildInputs = [ unzip xlibs.libX11 mesa ];
+
+  src = fetchurl {
+    url = "https://vorboss.dl.sourceforge.net/project/anttweakbar/AntTweakBar_116.zip";
+    sha256 = "0z3frxpzf54cjs07m6kg09p7nljhr7140f4pznwi7srwq4cvgkpv";
+  };
+
+  postPatch = "cd src";
+  installPhase = ''
+    mkdir -p $out/lib/
+    cp -vi ../lib/{libAntTweakBar.so,libAntTweakBar.so.1,libAntTweakBar.a} $out/lib/
+    cp -vi -r ../include $out/
+  '';
+
+  meta = {
+    description = "Add a light/intuitive GUI to OpenGL applications";
+    longDescription = ''
+      A small and easy-to-use C/C++ library that allows to quickly add a light
+      and intuitive graphical user interface into graphic applications based on OpenGL
+      (compatibility and core profiles), DirectX 9, DirectX 10 or DirectX 11
+      to interactively tweak parameters on-screen
+    '';
+    homepage = http://anttweakbar.sourceforge.net/doc/;
+    license = [ stdenv.lib.licenses.libpng stdenv.lib.licenses.zlib ];
```

Comment:
Maybe just call it zlib license? I have a feeling that license-as-a-list is not used consistently, and is confusing in some cases.

Reply:
I would prefer to keep both since that is what is specified by the author. But I'll remove it if you say it is ok.

Reply:
I think the point is that zlib and libpng is approximately the same license. 

https://opensource.org/licenses/Zlib

In case where two licenses make sense, a comment to explain the relation (we don't have a real consensus if it is «portions like this, portions like that» or «at your option») would be nice. But here you can just pick one.